mlrun/model.py

```diff
@@ -281,6 +378,18 @@ def __init__(
 
 
 class BaseMetadata(ModelObj):
+    _fields_to_strip = [
+        # "tag", # should tag be removed as well?
```

Comment:
My opinion is that `tag` needs to be removed, because it's only meaningful in the context of the current project and list of existing instances (it is a marker for a specific version of the object) - when importing it, the tag should be re-applied.